user-interface/src/main/resources/application.properties

```diff
@@ -2,6 +2,13 @@
 ################### Logging ###################################################
 logging.level.org.atmosphere=warn
 ###############################################################################
+################### Security ##################################################
+# Sets the environment variable that contains the vault key
```

Comment:
Can you move the description to the configuration key so it can be retrieved by the IDE? That would be great. IntelliJ helps out here with Alt+Enter or Option+Enter
Screenshot 2025-02-12 at 14.37.31.png